app/dts/bindings/zmk,layers.yaml

```diff
@@ -9,6 +9,7 @@ child-binding:
   properties:
     label:
       type: string
-    keys:
-      type: array
-
+      required: true
+    bindings:
```

Comment:
Renamed this to "bindings" since there's more than just "keys" in here, there's behaviors to bind to the key positions.

Thoughts on the naming?